src/service/RewardProgramService.ts

```diff
@@ -119,7 +119,7 @@ export class RewardProgramService implements TransactionFactory {
             return transactions;
         }
         const agentUrl = nodePreset.agentUrl || `https://${nodePreset.host}:7880`;
-        const plainMessage = `enrol ${agentPublicKey} ${agentUrl}`;
+        const plainMessage = `enroll ${agentPublicKey} ${agentUrl}`;
```

Comment:
Message should be in this format => `enroll <agent_url> <base64_agent_ca_csr_file>`

base64_agent_ca_csr_file => when decoded the text should include
` -----BEGIN CERTIFICATE REQUEST---xxx -----END CERTIFICATE REQUEST-----`